client/js/components/button/HamburgerMenuButton.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <!--The attribute data-responsive-menu-helper is used as a hook to append an EventListener in GlobalEventlisteners.js-->
+  <button
+    type="button"
+    class="btn btn--blank font-size-medium"
+    aria-haspopup="true"
+    aria-expanded="false"
+    :aria-label="Translator.trans('menu.nav.toggle')"
+    data-responsive-menu-helper>
+    <a class="nav-toggle">
```

Comment:
There is still the a in the b :)

Reply:
done https://github.com/demos-europe/demosplan-core/pull/1389/commits/7164868f2774114c952225ab2623e37a6aba2cf6